apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/PipelineApplicationIT.java

```diff
@@ -205,19 +206,20 @@ public void smallAmountOfRifData() throws IOException, InterruptedException {
               CcwRifLoadJob.S3_PREFIX_COMPLETED_DATA_SETS,
               new DataSetManifestEntry("beneficiaries.rif", RifFileType.BENEFICIARY),
               new DataSetManifestEntry("carrier.rif", RifFileType.CARRIER));
-      s3Client.putObject(DataSetTestUtilities.createPutRequest(bucket, manifest));
-      s3Client.putObject(
-          DataSetTestUtilities.createPutRequest(
-              bucket,
-              manifest,
-              manifest.getEntries().get(0),
-              StaticRifResource.SAMPLE_A_BENES.getResourceUrl()));
-      s3Client.putObject(
-          DataSetTestUtilities.createPutRequest(
-              bucket,
-              manifest,
-              manifest.getEntries().get(1),
-              StaticRifResource.SAMPLE_A_CARRIER.getResourceUrl()));
+      DataSetTestUtilities.putObject(s3Client, bucket, manifest);
```

Comment:
This seems a little weird; we have a single manifest that has 2 entries (BENEFICIARY and CARRIER). Not sure why we push the manifest twice; a single manifest can support 'n' entries (which is the case here) but we 'push' 2 manifests to the bucket. If this is intended, then maybe a comment to that effect would be appropriate.

Reply:
Right - I *did* push it twice (on accident) in a previous commit but that was before opening the PR. I'm only seeing it pushed once, and then the individual data files thereafter, like Brian is saying as well.

Reply:
> Right - I _did_ push it twice (on accident) in a previous commit but that was before opening the PR. I'm only seeing it pushed once, and then the individual data files thereafter, like Brian is saying as well.

I've taken an even closer look... and I agree! This isn't an issue any longer. I'm probably just using GitHub _wrong_–it was leading me to believe there were still two pushes of the manifest. Sorry for _my_ confusion, friends! Resolving...